.eslintrc.json

```diff
@@ -7,16 +7,52 @@
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
+        "plugin:tailwindcss/recommended",
```

Comment:
I used the tailwind eslint plugin to find the places where we used arbitrary classes etc